app.ini

```diff
@@ -15,3 +16,9 @@ processes = 2
 http = 0.0.0.0:5000
 vacuum = true
 die-on-term = true
+single-interpreter = true
+need-app = true
+
+disable-logging = true
```

Comment:
I'm not sure what's best here. When I'm dealing with "too much uneeded logs", I have to think hard of removing the logs is the fix or if you want to filter logs better. Any log aggregation framework will allow you to filter logs or ingest differently. 


If you just want to disable logging for healthz, you can probably use 
`route = ^/healthz donotlog:`

https://stackoverflow.com/a/25686557

My personal preferences are to find ways to better view logs and filter `/healthz` when interacting with them. That means when you need to inspect any 2xx, 3xx, logs you'll still have them.


If this is a cost optimization then by all means reduce the logging, otherwise I'd go with my recommendation. The choice is yours though. I think the other changes are a good.


I'd also add 
```
die-on-term = true  
vacuum = true
```

Reply:
Lines 17 and 18 have the `die-on-term` and `vacuum` settings you recommend, yeah?

Reply:
Thanks for the `donotlog` suggestion, I've been looking for that and couldn't find it. I think I'll go ahead and do that instead. To disable logging altogether isn't a huge deal though, because I explicitly log the POST and PUT requests when they come in, and I can't imagine wanting to see GET requests that are 2xx... The 3xx don't log right now anyway. Also of note, we're still on DEBUG level logging in production right now, but I'm fine keeping that until it gets unwieldy. I'll disable logging if and when we need to, but that healthz has got to go

Reply:
log level isn't really important until it is. Sounds good.